x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecks.java

```diff
@@ -169,6 +169,17 @@ static DeprecationIssue watcherNotificationsSecureSettingsCheck(Settings nodeSet
         return null;
     }
 
+    static DeprecationIssue watcherHipchatNotificationSettingsCheck(Settings nodeSettings, PluginsAndModules plugins) {
```

Comment:
in the above method `watcherNotificationsSecureSettingsCheck`, we should remove the hipchat check since the new one will superseded it.

Reply:
I removed the now unneeded check